rails/strings.rb

```diff
@@ -0,0 +1,1062 @@
+def strings
+    {
```

Comment:
moved every string here since reading a yml on every transformer run would impact performance quite heavily. This is a tradeoff to have strings in ruby file and require them from the transformer. In the long run we need to figure out strings management.